Seminar3/Task2.py

```diff
@@ -0,0 +1,15 @@
+from random import randint
+
+n = int(input('Введите число элементов в списке: '))
+lst = [randint(-10, 10) for i in range(n)]
+print(f'Заданный список:\n', lst)
+lst2 = []
+if len(lst) % 2 == 0:
+    for i in range(0, int(len(lst) / 2)):            
```

Comment:
```suggestion
    for i in range(n // 2):            
```
То же самое, не правда ли?